src/main/java/net/nikr/eve/jeveasset/data/api/accounts/AbstractOwner.java

```diff
@@ -61,6 +61,7 @@
 	private long ownerID;
 	private boolean showOwner = true;
 	private boolean invalid = false;
+	private MyAsset activeShip;

```

Comment:
After looking through the code, I think it's bad to use MyAsset here, sorry, I know I did agree to use MyAsset >_<
I think it would be better to create a new class to hold the data:
//Saved between restarts
locationID 
typeID
itemID
//Set when updating eventList
MyLocation
typeName
shipName
That way it's independent of actually assets. I really sorry I made you do it with MyAsset first

Reply:
I can do that. I think I was just hooking into the object that comes back from the request to add the asset into inventory. Should be fairly simple to just set it to something else.

Reply:
So after looking at this I'm not sure we need all 3 IDs to be save. Item will give you the location and type, so by having the itemID you can find both of the others.

Reply:
I feel like saving/loading two more values is an okay trade off for not having to iterate over the full asset list and having the data be independent of the assets.

Reply:
You still have to get the item to get the in game set Ship Name or save that as well.

Reply:
You can do itemID to shipName with:
`Settings.get().getEveNames()`

Reply:
Didn't realize that lookup existing, I'll use that then.

Reply:
Should all be implemented now.